example/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pinwheel-react-native-example",
-  "version": "3.1.1",
+  "version": "3.2.0",
```

Comment:
Can we revert this to `3.2.0-rc.5` and then merge it in, so we don't trigger a publish? There's at least one change (https://github.com/underdog-tech/react-native-pinwheel/pull/125) which we should include in the 3.2.0 release

Reply:
Is this addressed in [the commit](https://github.com/underdog-tech/react-native-pinwheel/pull/137/commits/db2402b968307f298b3098aa3ba3ea369d083dd4) which changes the package's version?